lib/adapter.js

```diff
@@ -1621,6 +1621,14 @@ function Adapter(options) {
                 callback && callback('Objects database not connected');
                 return;
             }
+            if (id === null || id === undefined || id === '') {
+                callback && callback('setObjectWithDefaultValue id missing');
```

Comment:
Can we have a more user/dev friendly error message please? It should be clear from the calling code that `setObjectWithDefaultValue` was called.

Reply:
I see this more for the devs ... because this should oly happen during development.
Ok, I can remove this, makes it easier to debug in my eyes becaue many people just log errors and then debugging is more complex